vpcmanager/config/machine.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Remove machines.json. Vpc Manager is supposed not to rely on physical topology therefore no need to import machine.json.